en/docs/updates/continuous-update.md

```diff
@@ -42,4 +44,5 @@ Refer the following diagram for a better understanding on continuous updates:
     </div>
 </div>
 
-Need to further refer WSO2 recommended Puppet and Ansible, Click [Here](../faq/#what-are-the-recommended-configuration-management-tools-to-deploy-configurations-to-client-nodes).
+!!! Note
+    Require additional references to WSO2's recommended Puppet and Ansible configurations. Click [Here](../faq/#what-are-the-recommended-configuration-management-tools-to-deploy-configurations-to-client-nodes).
```

Comment:
Shall we reword this like "For further insights into WSO2's recommended Puppet and Ansible configurations, please refer to [link]."